packages/web-components/fast-foundation/src/avatar/avatar.template.ts

```diff
@@ -0,0 +1,30 @@
+import { html, when } from "@microsoft/fast-element";
+import type { ViewTemplate } from "@microsoft/fast-element";
+import type { Avatar, AvatarOptions } from "./avatar";
+
+/**
+ * The template for {@link @microsoft/fast-foundation#Avatar} component.
+ * @public
+ */
+export const avatarTemplate: (
+    context,
+    definition: AvatarOptions
+) => ViewTemplate<Avatar> = (context, definition: AvatarOptions) => html`
+    <div
+        class="backplate ${x => x.shape}"
+        part="backplate"
+        style="${x =>
+            x.fill ? `background-color: var(--avatar-fill-${x.fill});` : void 0}"
+    >
+        <a
+            class="link"
+            part="link"
+            href="${x => (x.link ? x.link : void 0)}"
+            style="${x => (x.color ? `color: var(--avatar-color-${x.color});` : void 0)}"
+        >
+            <slot name="media" part="media">${definition.media || ""}</slot>
+            <span class="name" part="name">${x => x.initials}</span>
```

Comment:
In relation to my comment below about the relationship between `initials` and `name`, this feels odd to me, that we call this part `name` but use `initials`. Why does the component accept both pieces of data if it only really uses `initials`?

@chrisdholt It would be good to get your thoughts here, but my thinking is that we probably should just accept content in the form of a default slot and leave the content formatting up to app authors.

Reply:
I'm inclined to agree, after spending more time with web components since initially writing this, it does seem like bad form. I went ahead and removed both `initials` and `name` from `avatar` in favor of using the default slot for text content.